nginx/setup.sh

```diff
@@ -69,5 +69,4 @@ if [ "$1" != "proxy-*.conf" ] ; then
         rm -rf /etc/nginx/sites-enabled/default || echo exit 8
         rm -rf /etc/nginx/sites-enabled/000-default-le-ssl-local.conf || exit 9
         ln -s /etc/nginx/sites-available/000-default-le-ssl-local.conf /etc/nginx/sites-enabled/000-default-le-ssl-local.conf || exit 10
-fi
-
+fi
```

Comment:
OK to remove line 73, but 72 must end with a newline.

Reply:
It's done.